release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -292,8 +292,13 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
 [id="ocp-4-10-monitoring"]
 === Monitoring
 
-[id="ocp-4-10-scale"]
-=== Scale
+[id="ocp-4-10-scalability-and-performance"]
+=== Scalability and performance
+
+==== New Special Resource Operator metrics
```

Comment:
Your L4 heading needs a unique id.

Something like this:

`[id="ocp-4-10-SRO-metrics]`

Reply:
OH I didn't realize that was a thing. Will add, thank you :)